authority/provisioner/wire/oidc_options.go

```diff
@@ -0,0 +1,155 @@
+package wire
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"text/template"
+	"time"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+	"go.step.sm/crypto/x509util"
+)
+
+type Provider struct {
+	IssuerURL   string   `json:"issuer,omitempty"`
+	AuthURL     string   `json:"authorization_endpoint,omitempty"`
+	TokenURL    string   `json:"token_endpoint,omitempty"`
+	JWKSURL     string   `json:"jwks_uri,omitempty"`
+	UserInfoURL string   `json:"userinfo_endpoint,omitempty"`
+	Algorithms  []string `json:"id_token_signing_alg_values_supported,omitempty"`
+}
+
+type Config struct {
+	ClientID                   string           `json:"clientId,omitempty"`
+	SignatureAlgorithms        []string         `json:"signatureAlgorithms,omitempty"`
```

Comment:
I don't think this is necessary anymore. It's also a property that is usually in the JWKs and on the `id_token_signing_alg_values_supported` property of the openid-configuration. And you have those in the provider.

Reply:
That leaves us with only the ClientID. Assuming the rest of fields are for testing, having something like this makes more sense to me:
```go
type OIDCOptions struct {
	ClientID          string    `json:"clientId,omitempty"` // omitempty?
	Provider          *Provider `json:"provider,omitempty"`
	TransformTemplate string    `json:"transform,omitempty"`
}
```